Readme.md

```diff
@@ -0,0 +1,60 @@
+**Pre-requisite:**
+
+1. python2
```

Comment:
Right, nginx is being used as the backend. However, nginx is being installed automatically by our script on the Virtual Machine in which we run our experiments.
Yes, we need to say that `lyft/envoy`'s binary is needed to provided for the experiments. I'll add that.